### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by ocefpaf**
This one is ready but it is best merged after https://github.com/ioos/glider-dac/pull/212 so we can test the GHA here.

- [x] 1. Status 0 seems to be related to https://github.com/tcort/markdown-link-check/issues/220
- [x] 2. Status 400 are local references that exists but the checker is failing somehow
- [x] 3. Fix all the references to `kerfoot` pages
- [x] 4. Re-check the changed links to ensure they are showing the expected info from before

  ERROR: 9 dead links found in _docs/glider-background-and-sampling-terminology.md !
  [✖] https://apl.uw.edu/project/project.php?id=seaglider#:~:text=The → Status: 0
  [✖] https://github.com/kerfoot/glider-dac/blob/master/nc/template/IOOS_Glider_NetCDF_v2.0.cdl → Status: 404
  [✖] https://github.com/kerfoot/glider-dac/blob/master/nc/template/IOOS_Glider_NetCDF_v2.0.nc → Status: 404
  [✖] https://github.com/kerfoot/glider-dac/blob/master/nc/template/IOOS_Glider_NetCDF_v2.0.ncml → Status: 404
  [✖] https://github.com/kerfoot/glider-dac/tree/master/nc/template → Status: 404
  [✖] ngdac-netcdf-file-format-version-2.html → Status: 400
  [✖] /glider-dac/glider-sampling-terminology.png → Status: 400

  ERROR: 3 dead links found in _docs/index.md !
  [✖] /glider-dac/ngdac-netcdf-file-format-version-2.html → Status: 400
  [✖] glider-background-and-sampling-terminology.html#sampling-pattern-terminology → Status: 400

  ERROR: 6 dead links found in _docs/ngdac-netcdf-file-format-version-2.md !
  [✖] /glider-dac/ngdac-netcdf-file-submission-process.html → Status: 400
  [✖] https://github.com/ioos/glider-dac/blob/master/nc/template/IOOS_Glider_NetCDF_v2.0.nc → Status: 404
  [✖] https://github.com/ioos/glider-dac/blob/master/nc/template/IOOS_Glider_NetCDF_v2.0.cdl → Status: 404
  [✖] https://github.com/ioos/glider-dac/blob/master/nc/template/IOOS_Glider_NetCDF_v2.0.ncml → Status: 404
  [✖] https://github.com/ioos/glider-dac/tree/master/nc/template → Status: 404

  ERROR: 5 dead links found in _docs/useful-links.md !
  [✖] ngdac-netcdf-file-format-version-2.html → Status: 400
  [✖] https://compliance.ioos.us/index.html → Status: 502
  [✖] https://github.com/ioos/glider-dac/blob/master/util/ncFtp2ngdac.pl → Status: 404
  [✖] /glider-dac/ngdac-netcdf-file-submission-process.html#new-deployment-registration → Status: 400

  ERROR: 1 dead links found in _docs/data-workflow.md !
  [✖] https://www.ndbc.noaa.gov/gliders.php → Status: 404

  ERROR: 7 dead links found in _docs/glider-dac-communications-plan.md !
  [✖] https://github.com/ioos/glider-dac-catalog → Status: 404
  [✖] https://github.com/ioos/glider_dac_thredds_catalog%3E → Status: 404
  [✖] http://www.ioos.noaa.gov/observing/observing_assets/glider_asset_map.html → Status: 404
  [✖] https://gliders.ioos.us/metadata-erddap/ → Status: 404
  [✖] https://gliders.ioos.us/metadata-thredds/ → Status: 404
  [✖] http://www.ndbc.noaa.gov/gliders.php → Status: 404
  [✖] https://raw.githubusercontent.com/kerfoot/glider-dac/master/doco/IOOS-DAC-architecture.png → Status: 404

  ERROR: 1 dead links found in _docs/ngdac-architecture.md !
  [✖] /glider-dac/IOOS-DAC-architecture.png → Status: 400

  ERROR: 11 dead links found in _docs/ngdac-netcdf-file-submission-process.md !
  [✖] /glider-dac/useful-links → Status: 400
  [✖] ngdac-netcdf-file-format-version-2.html#description--examples-of-required-global-attributes → Status: 400
  [✖] ngdac-netcdf-file-format-version-2.html#platform → Status: 400
  [✖] ngdac-netcdf-file-format-version-2.html#trajectory → Status: 400
  [✖] ngdac-netcdf-file-submission-process.html#dataset-archiving → Status: 400
  [✖] ngdac-netcdf-file-format-version-2.html#file-naming-conventions → Status: 400
  [✖] https://github.com/ioos/glider-dac/blob/master/util/ncFtp2ngdac.pl → Status: 404
  [✖] /glider-dac/ngdac-architecture.html#data-assembly-center-architecture → Status: 400
  [✖] https://compliance.ioos.us/index.html → Status: 502
  [✖] /glider-dac/DAC_providers_your_deployments.png → Status: 400
  [✖] /glider-dac/DAC_complete_deployment.png → Status: 400

### CODE REVIEW

**Comment by ocefpaf** [comment]
The https://compliance.ioos.us/index.html broken link is on us:

Screenshot from 2023-08-18 20-41-28

**Comment by ocefpaf** [comment]
For 3 we need to check:

```
find . -type f -name "*.md" -exec grep "kerfoot" '{}' \; -print  | grep \.\/    
![DAC ARchitecture](https://raw.githubusercontent.com/kerfoot//glider-dac//master/doco/IOOS-DAC-architecture.png)
./glider-dac-communications-plan.md
> [Wiki](https://github.com/kerfoot/ioosngdac/wiki) ▸ **Glider Background and Sampling Terminology**
<!-- ![Glider Sampling Patterns and Terms](https://raw.githubusercontent.com/kerfoot/ioosngdac/master/doco/glider-sampling-terminology.png) -->
 + **Profile**: A single vertically oriented track of a glider, either upward or downward through the water column.  A profile is one-half of a **dive**.  The profile is the fundamental atomic data type used by the **NGDAC**.  All data submitted to the **NGDAC** is submitted as individual profiles, containing the various water column properties or sensor values.  Examples of the file format description can be found as [CDL](https://github.com/kerfoot/ioosngdac/blob/master/nc/template/IOOS_Glider_NetCDF_v2.0.cdl), [NetCDF file](https://github.com/kerfoot/ioosngdac/blob/master/nc/template/IOOS_Glider_NetCDF_v2.0.nc) and [ncml](https://github.com/kerfoot/ioosngdac/blob/master/nc/template/IOOS_Glider_NetCDF_v2.0.ncml) can be found [here](https://github.com/kerfoot/ioosngdac/tree/master/nc/template).
./glider-background-and-sampling-terminology.md
> [Wiki](https://github.com/kerfoot/glider-dac/wiki) ▸ **Data Provider Links**
./useful-links.md
> [Wiki](https://github.com/kerfoot/ioosngdac/wiki) ▸ **NGDAC File Submission Process**
./ngdac-netcdf-file-submission-process.md
> [Wiki](https://github.com/kerfoot//glider-dac//wiki) ▸ **NGDAC Architecture**
./ngdac-architecture.md
```

File: _docs/data-workflow.md [hunk 1]
```diff
@@ -15,16 +15,16 @@ One function of the Glider DAC is to coordinate with the NWS' National Data Buoy
...
- 2. **Do they transition to historical as soon as they are recovered or is there a different mechanism to transition them?**             
+ 1. **Do they transition to historical as soon as they are recovered or is there a different mechanism to transition them?**             
  A glider is no longer be classified as active if new data has not been available for five days.  This is automatically done by the web processing software.
 
 ### The Observing System Monitoring Center (OSMC)
 
-[OSMC](www.osmc.info) is a monitoring tool that attempts to harvest and track all ocean observations on the GTS.  This tool is comprehensive but not perfect so if discrepancies are found we encourage users to contact OSMC directly OSMC.Webmaster-AT-noaa-DOT-gov.
+[OSMC](https://www.osmc.noaa.gov/) is a monitoring tool that attempts to harvest and track all ocean observations on the GTS.  This tool is comprehensive but not perfect so if discrepancies are found we encourage users to contact OSMC directly OSMC.Webmaster-AT-noaa-DOT-gov.
```
Comment (ocefpaf) [inline _docs/data-workflow.md:27]: @kbailey-noaa some links, like this one, I'm not 100% sure the new one is the right substitute for the dead one.

File: _docs/faqs.md [hunk 2]
```diff
@@ -37,4 +37,4 @@ The National Data Buoy Center harvests new profile observations from the [DAC's
 
 <!-- 2. Can I submit raw glider files to the DAC? -->
 
-<!-- At the moment no, you must convert the raw files to netCDF following the DAC [metadata conventions](/ioosngdac/ngdac-netcdf-file-format-version-2.html). But we're in the process of developing a raw data upload tool that will enable users to  -->
+<!-- At the moment no, you must convert the raw files to netCDF following the DAC [metadata conventions](/glider-dac/ngdac-netcdf-file-format-version-2.html). But we're in the process of developing a raw data upload tool that will enable users to  -->
```
Comment (ocefpaf) [inline _docs/faqs.md:40]: Everything here is commented out, no text is created for the page, should we clean this up? (If yes, let's do in another PR to avoid cluttering this one.)

File: _docs/glider-background-and-sampling-terminology.md [hunk 3]
```diff
@@ -20,18 +20,18 @@ summary: This page defines glider and sampling terminology used throughout the r
...
- + [Spray](http://spray.ucsd.edu/pub/rel/info/spray_description.php): originally designed by [Scripps Institution of Oceanography](https://scripps.ucsd.edu/) and [Woods Hole Oceangraphic Institution](http://www.whoi.edu/) with funding provided by the [Office of Naval Research](http://www.onr.navy.mil/), the Spray glider is now manufactured by [Bluefin Robotics](http://www.bluefinrobotics.com/products/spray-glider/).
- + [Slocum](http://www.webbresearch.com/slocumglider.aspx): designed and built by [Teledyne Webb Research Corporation](http://www.webbresearch.com/).
+ + [Seaglider](https://apl.uw.edu/project/project.php?id=seaglider#:~:text=The AUV Seaglider is the,time via satellite data telemetry.): originally designed and built through a collaboration with the University of Washington's [Applied Physics Lab](https://apl.uw.edu/) and [School of Oceanography](https://www.ocean.washington.edu/).  The Seaglider is now manufactured by [Hydroid](https://hii.com/what-we-do/capabilities/unmanned-systems/)
```
Comment (ocefpaf) [inline _docs/glider-background-and-sampling-terminology.md:23]: The Hydroid used to point to a seaglider page. I could not find any mentions to seaglider in their new page. The old one redirects to https://hii.com/what-we-do/capabilities/unmanned-systems/.

File: _docs/glider-background-and-sampling-terminology.md [hunk 4]
```diff
@@ -20,18 +20,18 @@ summary: This page defines glider and sampling terminology used throughout the r
...
+ + [Spray](https://spray.ucsd.edu/pub/rel/info/spray_description.php): originally designed by [Scripps Institution of Oceanography](https://scripps.ucsd.edu/) and [Woods Hole Oceangraphic Institution](http://www.whoi.edu/) with funding provided by the [Office of Naval Research](https://www.nre.navy.mil/), the Spray glider is now manufactured by [Bluefin Robotics](https://gdmissionsystems.com/underwater-vehicles/bluefin-robotics/).
```
Comment (ocefpaf) [inline _docs/glider-background-and-sampling-terminology.md:24]: The old ONR is dead and this new NRE one seems to be the best fit. The bluefin robotics also changed the domain, not sure if the new page is the right subtitute.

File: _docs/glider-background-and-sampling-terminology.md [hunk 5]
```diff
@@ -20,18 +20,18 @@ summary: This page defines glider and sampling terminology used throughout the r
...
+ + **Profile**: A single vertically oriented track of a glider, either upward or downward through the water column.  A profile is one-half of a **dive**.  The profile is the fundamental atomic data type used by the **NGDAC**.  All data submitted to the **NGDAC** is submitted as individual profiles, containing the various water column properties or sensor values.  Examples of the file format description can be found as [CDL](https://github.com/kerfoot/glider-dac/blob/master/nc/template/IOOS_Glider_NetCDF_v2.0.cdl), [NetCDF file](https://github.com/kerfoot/glider-dac/blob/master/nc/template/IOOS_Glider_NetCDF_v2.0.nc) and [ncml](https://github.com/kerfoot/glider-dac/blob/master/nc/template/IOOS_Glider_NetCDF_v2.0.ncml) can be found [here](https://github.com/kerfoot/glider-dac/tree/master/nc/template).
```
Comment (ocefpaf) [inline _docs/glider-background-and-sampling-terminology.md:34]: These are problematic b/c they are links to John's fork. Should we transfer those files here and fix these links? Should we review/re-write this to be sure we still need this?

File: _docs/glider-dac-communications-plan.md [hunk 6]
```diff
@@ -19,7 +19,7 @@ Currently we have the following URLs that contain some piece of the glider story
 # Technical Docs and Github Repos
```
Comment (ocefpaf) [inline _docs/glider-dac-communications-plan.md:19]: This whole page will be gone when merge https://github.com/ioos/glider-dac/pull/261. I would not bother reviewing it.

File: _docs/index.md [hunk 7]
```diff
@@ -30,12 +30,12 @@ The goals of the <b>U.S. IOOS National Glider Data Assembly Center</b>:
  + Develop a simple, fully self-describing [NetCDF](http://en.wikipedia.org/wiki/NetCDF) file specification that preserves the resolution of the original glider data sets.
  + Provide glider operators with a simple process for registering and submitting glider data sets to a centralized storage location.
  + Provide public access to glider data sets via existing web services and standards, in a variety of well-known formats.
- + Facilitate the distribution of glider data sets on the [Global Telecommunication System](http://www.wmo.int/pages/prog/www/TEM/GTS/index_en.html).
+ + Facilitate the distribution of glider data sets on the [Global Telecommunication System](https://public.wmo.int/en/programmes/global-telecommunication-system).
```
Comment (ocefpaf) [inline _docs/index.md:33]: Again, tried to find the best fit for the old link but I'm not 100% sure.

File: _docs/data-workflow.md [hunk 8]
```diff
@@ -15,16 +15,16 @@ One function of the Glider DAC is to coordinate with the NWS' National Data Buoy
 
 ### NDBC Glider Page
 
-Active and historical glider deployments are posted to an [NDBC glider page](http://www.ndbc.noaa.gov/gliders.php).  These questions and answers from NDBC clarify the information on the glider page specific information and GTS.  
+Active and historical glider deployments are posted to an [NDBC glider page](https://www.ndbc.noaa.gov/gliders.php).  These questions and answers from NDBC clarify the information on the glider page specific information and GTS.
```
Comment (ocefpaf) [inline _docs/data-workflow.md:18]: https://www.ndbc.noaa.gov/gliders.php is a NDBC 404 page! I cannot find a replacement for this one.

File: _docs/data-workflow.md (hunk 1 above)
Comment (kbailey-noaa) [reply _docs/data-workflow.md:27]: @ocefpaf the content in this .md file is obsolete (e.g. NDBC doesn't publish glider data anymore) and I don't think this .md is even linked in the sidebar/published to https://ioos.github.io/glider-dac/.  Recommend removing (or similar action to glider-dac-communications-plan.md).

File: _docs/faqs.md (hunk 2 above)
Comment (kbailey-noaa) [reply _docs/faqs.md:40]: I'd leave commented out, but it can't hurt to move forward with that link repair.

File: _docs/data-workflow.md (hunk 1 above)
Comment (ocefpaf) [reply _docs/data-workflow.md:27]: Done!

File: _docs/glider-background-and-sampling-terminology.md (hunk 3 above)
Comment (kbailey-noaa) [reply _docs/glider-background-and-sampling-terminology.md:23]: @ocefpaf Hydroid is no longer servicing Seagliders.  Change the first Seaglider link to https://apl.uw.edu/project/project.php?id=seaglider (not sure why the link there now results in highlighted text).  Just delete the sentence "The Seaglider is now manufactured by..."

File: _docs/glider-background-and-sampling-terminology.md (hunk 5 above)
Comment (kbailey-noaa) [reply _docs/glider-background-and-sampling-terminology.md:33]: Yes, transfer files and fix links.  Documents shouldn't be stored in forks.

File: _docs/index.md [hunk 9]
```diff
@@ -30,12 +30,12 @@ The goals of the <b>U.S. IOOS National Glider Data Assembly Center</b>:
...
+Transmission of the data sets via the [Global Telecommunication System](https://public.wmo.int/en/programmes/global-telecommunication-system) is made possible by the [National Data Buoy Center](http://www.ndbc.noaa.gov/), which accesses the data sets from the **ERDDAP** and/or **THREDDS** end-points. After performing some internal quality checks, the profiles are encoded into [BUFR](http://en.wikipedia.org/wiki/BUFR) format and released on the [Global Telecommunication System](https://public.wmo.int/en/programmes/global-telecommunication-system), making them available for assimilation by regional and global scale ocean forecasting models.
 
 Please read the documentation **thoroughly** before beginning the data submission process.  Additional questions and information requests should be directed to:
```
Comment (kbailey-noaa) [inline _docs/index.md:40]: Delete the sentence "Please read documentation **thoroughly**..."

File: _docs/glider-background-and-sampling-terminology.md (hunk 3 above)
Comment (ocefpaf) [reply _docs/glider-background-and-sampling-terminology.md]: Done!

File: _docs/glider-background-and-sampling-terminology.md (hunk 5 above)
Comment (ocefpaf) [reply _docs/glider-background-and-sampling-terminology.md]: Done!

File: _docs/index.md (hunk 9 above)
Comment (ocefpaf) [reply _docs/index.md]: Done!

File: _docs/index.md (hunk 7 above)
Comment (ocefpaf) [reply _docs/index.md]: Done! (I change all 3 occurrences for this link.)

File: _docs/useful-links.md [hunk 10]
```diff
@@ -10,7 +10,7 @@ summary: A list of links for Data Providers
 ---
 
 <!--
-> [Wiki](https://github.com/kerfoot/ioosngdac/wiki) ▸ **Data Provider Links**
+> [Wiki](https://github.com/kerfoot/glider-dac/wiki) ▸ **Data Provider Links**
```
Comment (kbailey-noaa) [inline _docs/useful-links.md:13]: I see this is commented out, but given Derrick's guidance that we are not to use the repo Wiki, we'll need to remove this sentence entirely.

File: _docs/useful-links.md [hunk 11]
```diff
@@ -29,42 +29,36 @@ The following is a list of links for data providers and web applications develop
 
   The IOOS Compliance Checker is a python based tool for data providers to check for completeness and community standard compliance of local or remote netCDF files against metadata standards. This web based tool can be used to check your GliderDAC netCDF files for compliance. Be sure to select **GliderDAC** in the list of available tests to run.
 
- - [**FTP script**](https://github.com/ioos/glider-dac/blob/master/util/ncFtp2ngdac.pl)
```
Comment (ocefpaf) [inline _docs/useful-links.md]: @benjwadams can you give your :+1: or :-1:  here? As far as I remember this is deprecated, right?

File: _docs/useful-links.md [hunk 12]
```diff
@@ -29,7 +29,7 @@ The following is a list of links for data providers and web applications develop
 
   The IOOS Compliance Checker is a python based tool for data providers to check for completeness and community standard compliance of local or remote netCDF files against metadata standards. This web based tool can be used to check your GliderDAC netCDF files for compliance. Be sure to select **GliderDAC** in the list of available tests to run.
 
- - [**FTP script**](https://github.com/ioos/ioosngdac/blob/master/util/ncFtp2ngdac.pl)
+ - [**FTP script**](https://github.com/ioos/glider-dac/blob/master/util/ncFtp2ngdac.pl)
```
Comment (kbailey-noaa) [inline _docs/useful-links.md]: @benjwadams @mwengren FYI - this .pl script includes data.ioos.us instead of gliders.ioos.us, and is linked in the User Documentation (https://ioos.github.io/glider-dac/useful-links.html).  Will this script be updated?  Or should we remove it?

Comment (kbailey-noaa) [reply _docs/useful-links.md]: Update - we'll remove this from the docs.

File: _docs/useful-links.md [hunk 13]
```diff
@@ -29,7 +29,7 @@ The following is a list of links for data providers and web applications develop
...
 
  For automating uploads of [conforming](ngdac-netcdf-file-format-version-2.html) NetCDF files to the DAC ftp server.  The script is written in Perl and requires the following non-core modules:
     + [Readonly](http://search.cpan.org/~roode/Readonly-1.03/Readonly.pm)
```
Comment (kbailey-noaa) [inline _docs/useful-links.md]: Remove or comment out rows 34 -35

File: _docs/useful-links.md [hunk 14]
```diff
@@ -39,13 +39,13 @@ The following is a list of links for data providers and web applications develop
 ## Submitting Data
  - [**Deployment Registration**](https://gliders.ioos.us/providers)
 
-This link will redirect you to the GliderDAC providers page. From there you can login and create deployments. See the wiki section on [New Deplyment Registration](/ioosngdac/ngdac-netcdf-file-submission-process.html#new-deployment-registration) for more details.
+This link will redirect you to the GliderDAC providers page. From there you can login and create deployments. See the wiki section on [New Deplyment Registration](/glider-dac/ngdac-netcdf-file-submission-process.html#new-deployment-registration) for more details.
```
Comment (kbailey-noaa) [inline _docs/useful-links.md]: We can't point to a separate wiki located elsewhere.  Also there's a typo in 'deployment.'
Change the sentence "See the wiki..." to: "See the section on [New Deployment Registration](https://ioos.github.io/glider-dac/ngdac-netcdf-file-submission-process.html#new-deployment-registration) for more details.

File: nc/template/IOOS_Glider_NetCDF_v3.0-noqartod.cdl
Comment (kbailey-noaa) [inline nc/template/IOOS_Glider_NetCDF_v3.0-noqartod.cdl:1]: Is this linked somewhere?

File: _docs/useful-links.md
Comment (kbailey-noaa) [inline _docs/useful-links.md:1]: Also, change row 55 - [**Dataset Status**](https://marine.rutgers.edu/cool/data/gliders/dac/status/)
 to [**Dataset Status**](https://gliders.ioos.us/status/)

**Review by kbailey-noaa — CHANGES_REQUESTED** [review]
Numerous comments throughout about link changes and some minor text updates.

File: _docs/ngdac-architecture.md [hunk 15]
```diff
@@ -10,7 +10,7 @@ summary: This page presents an in-depth description and discussion of the U.S. I
 ---
 
 <!--
-> [Wiki](https://github.com/kerfoot/ioosngdac/wiki) ▸ **NGDAC Architecture**
+> [Wiki](https://github.com/kerfoot/glider-dac/wiki) ▸ **NGDAC Architecture**
```
[the remark this answers is not in the document]
Comment (ocefpaf) [reply _docs/ngdac-architecture.md:13]: This is commented out code. I'd rather we remove them all b/c they are all over the place and most of them no longer makes sense.

File: _docs/index.md [hunk 16]
```diff
@@ -30,12 +30,12 @@ The goals of the <b>U.S. IOOS National Glider Data Assembly Center</b>:
  + Develop a simple, fully self-describing [NetCDF](http://en.wikipedia.org/wiki/NetCDF) file specification that preserves the resolution of the original glider data sets.
```
[the remark this answers is not in the document]
Comment (ocefpaf) [reply _docs/index.md:30]: Done!

File: _docs/ngdac-netcdf-file-format-version-2.md [hunk 17]
```diff
@@ -94,7 +94,7 @@ Ideally, the <strong>glider_yyyymmddTHHMMSS.nc</strong> files will be provided b
 ## Global Attributes
 
 The following is the list of required global attributes that must be included in each NetCDF file submitted to the **NGDAC**. This list was created from a variety of sources with the goal of providing a complete metadata record of the data set. More information on these sources can be found at the following locations:
- - **CF1.6**: Section 2.6 of v1.6 Climate and Forecast [conventions](http://cfconventions.org/cf-conventions/v1.6.0/cf-conventions.html)
+ - **CF1.6**: Section 2.6 of v1.6 Climate and Forecast [conventions](https://cfconventions.org/cf-conventions/v1.6.0/cf-conventions.html
```
[the remark this answers is not in the document]
Comment (ocefpaf) [reply _docs/ngdac-netcdf-file-format-version-2.md:48]: Eagle eye on that parenthesis! Done.

File: nc/template/IOOS_Glider_NetCDF_v3.0-noqartod.cdl
Comment (ocefpaf) [reply nc/template/IOOS_Glider_NetCDF_v3.0-noqartod.cdl:1]: There is a "catch-all" link in https://github.com/ioos/glider-dac/blob/eb94b60649589c01f032c222f49889913d61ed76/_docs/ngdac-netcdf-file-format-version-2.md?plain=1#L65 pointing to the folder. So I guess that we needed all the files in that folder.
